Backend/routes/listings.js

```diff
@@ -7,9 +7,11 @@ const cloudinary = require("../config/cloudinaryConfig");
 const { buildFilters } = require("../utils/buildFilters.js");
 const { sendNotification } = require("../services/notificationService");
 const NotificationType = require("../Enums/NotificationType.js");
+const logActivity = require("../middlewares/logActivity");
 
 const prisma = new PrismaClient();
 const router = express.Router();
+router.use(verifyToken);
```

Comment:
what does this do? do you still have to call verifyToken in the below functions then?

Reply:
No i don't, it already applies to all the routes. Made the fix